openslides/motions/views.py

```diff
@@ -395,8 +395,8 @@ def destroy(self, *args, **kwargs):
         """
         Customized view endpoint to delete a motion poll.
         """
-        result = super().destroy(*args, **kwargs)
         poll = self.get_object()
+        result = super().destroy(*args, **kwargs)
```

Comment:
@jwinzer @normanjaeckel Why we have to change the order here? Is it really required?